bog_app/app/views/creatures/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= image_tag @creature.avatar.url(:medium) %>
+<h2> <%= @creature.name %> </h2>
+<h3> <%= @creature.description %> </h3>
+<%= button_to "DELETE THIS MONSTER, YOU MONSTER", @creature, method: :delete %>
```

Comment:
Instead of manually setting the button's method to delete, I'd suggest using a link_to tag and adding a the class 'btn' since you're using Bootstrap. It's better to manipulate the appearance of a link rather than the behavior of a button.